lib/models/ui/user_profile_details.dart

```diff
@@ -1,13 +1,12 @@
+import "package:proxima/models/auth/auth_logged_in_user.dart";
 import "package:proxima/models/database/user/user_firestore.dart";
-import "package:proxima/models/login_user.dart";
 
-// TODO (model team) convert to freezed
 // TODO see with UI team what will be displayed for profile
```

Comment:
Isn't it solved? Everyone in the team seems to have approved of the current UI. 
Else I couldn't find any related task in the backlog, we could create it.

Reply:
Yes ty, done in 92a8a12